backend/api/submissions/forms.py

```diff
@@ -119,12 +119,13 @@ def save(self, commit=True):
         self.instance.speaker_id = request.user.id
         instance = super().save(commit=commit)
         notify_new_submission(
-            instance.title,
-            instance.elevator_pitch,
-            instance.type.name,
-            request.build_absolute_uri(instance.get_admin_url()),
-            instance.duration.duration,
-            instance.topic.name,
+            submission_id=instance.id,
+            title=instance.title,
+            elevator_pitch=instance.elevator_pitch,
+            submission_type=instance.type.name,
+            admin_url=request.build_absolute_uri(instance.get_admin_url()),
```

Comment:
this returns the wrong URL on staging, but on production should be correct

(this will be fixed once we fix the hack we have to force pycon.it)